parts/k8s/kuberneteswinagentresourcesvmas.t

```diff
@@ -272,7 +272,7 @@
       ],
       "properties": {
         "publisher": "Microsoft.AKS",
-        "type": "Compute.AKS-Engine.Windows.Billing",
+        "type": {{if IsHostedMaster}}"Compute.AKS.Windows.Billing"{{else}}"Compute.AKS-Engine.Windows.Billing"{{end}},
```

Comment:
I think the reason we don't need this is because hosted masters don't support Windows

(if this is helping the no template code this PR makes sense but otherwise this if statement is superfluous).

Reply:
/hold